superglue_ros/Keypoint[] ref_keypoints
superglue_ros/Keypoint[] cur_keypoints
float64[] match_score